[help wanted] asTerm called on not-a-Term val

while `migrate PROJECTID` it run a while until it ends with

> exception caught when loading class Deprecated: java.lang.AssertionError: assertion failed: asTerm called on not-a-Term val   \<none\>
> [error] 
> [error] 
> [error] Migration has failed
> [error] because of assertion failed: asTerm called on not-a-Term val <none>

I need help to get the file which cause that Exception.

--- a/plugin/src/main/scala/migrate/ScalaMigratePlugin.scala
+++ b/plugin/src/main/scala/migrate/ScalaMigratePlugin.scala
@@ -187,9 +187,11 @@ object ScalaMigratePlugin extends AutoPlugin {
         val scalaVersion         = Keys.scalaVersion.value
         val sOptions             = scalacOptions.value
         val classpath            = dependencyClasspath.value.map(_.data.toPath())
+        val scala3Lib            = scalaInstance.value.libraryJars.toSeq.map(_.toPath)
         val scala3ClassDirectory = (compile / classDirectory).value.toPath
         val scalac3Options       = sanitazeScala3Options(sOptions)
-        val scala3Inputs         = Scala3Inputs(projectId, scalaVersion, scalac3Options, classpath, scala3ClassDirectory)
+        val scala3Inputs =
+          Scala3Inputs(projectId, scalaVersion, scalac3Options, scala3Lib ++ classpath, scala3ClassDirectory)
         StateTransform(s => s.put(scala3inputsAttirbute, scala3Inputs))
       },
       storeScala2Inputs := {

